src/lib/OpenEXRCore/compression.c

```diff
@@ -119,12 +148,10 @@ exr_result_t exr_uncompress_buffer (
 
         if (res == LIBDEFLATE_SUCCESS)
         {
-            if (in_bytes == actual_in_bytes)
-                return EXR_ERR_SUCCESS;
+            if (in_bytes == actual_in_bytes) return EXR_ERR_SUCCESS;
```

Comment:
do these single line modifications play nice with our clang format file?

Reply:
I was running clang-format to make these changes. I hope it picked up our clang-format config :-)